database.json

```diff
@@ -868,6 +868,12 @@
     "repo": "denoget",
     "desc": "denoget installs executable deno script."
   },
+  "deno-hoarders": {
+    "type": "github",
+    "owner": "searchableguy",
+    "repo": "deno-hoarders",
+    "desc": "A simple utility to import every external deno module from deno/x/ 🚒."
+  },
```

Comment:
database names with dashes are not allowed so can change the name to deno_hoarders, denoHoardersor something else